README.md

```diff
@@ -1 +1,65 @@
-"# Leaderboard" 
+# To-do list
```

Comment:
- [ ] Kindly change the project title name into your README file since that one is a  Leaderboard and not To-do list project